infra/base-images/base-clang/Dockerfile

```diff
@@ -18,16 +18,23 @@
 
 FROM gcr.io/oss-fuzz-base/base-image
 
+ARG introspector
```

Comment:
I wonder if putting this so high up impacts caching. My first guess is that it won't but I think it's worth testing.